components/help-dropdown/help-dropdown.scss

```diff
@@ -0,0 +1,11 @@
+.custom-select {
+  display: block;
```

Comment:
Just out of curiosity: what's block doing for you that inline-block wasn't?

Reply:
removed that...literally it's doing nothing extra.